src/codegen/enums.rs

```diff
@@ -6,6 +6,7 @@ use crate::{
         self, cfg_condition, cfg_condition_no_doc, cfg_condition_string, cfg_deprecated, derives,
         doc_alias, version_condition, version_condition_no_doc, version_condition_string,
     },
+    codegen::object::generate_default_impl,
```

Comment:
Why do you move it to the `object` module? Has nothing to do with objects :)

Reply:
Where should I put it then?

Reply:
`codegen/mod.rs`? And maybe call it `generate_enum_flags_default_impl`

Reply:
Oh you only want it for enums and flags? Not for structs?

Reply:
How would this work properly for structs?

Reply:
Exactly the same. It's expecting an expression to put into the `default` function.

Reply:
Ah that is not really what I was thinking of, it should take a member name and check if it exists or error out. Then return `Self::MemberName` in the default impl.

Reply:
Hum ok. I'll send an error if objects have this attribute then.